docs/download.html

```diff
@@ -73,9 +74,9 @@ <h3>Download Constellation</h3>
                                 <li>Constellation Adapters</li>
```

Comment:
Adaptors

Reply:
Fixed thank you very much for picking up on the typo